sklearn/manifold/_classical_mds.py

```diff
@@ -0,0 +1,198 @@
+"""
+Classical multi-dimensional scaling (classical MDS).
+"""
+
+# Authors: The scikit-learn developers
+# SPDX-License-Identifier: BSD-3-Clause
+
+from numbers import Integral
+
+import numpy as np
+from scipy import linalg
+
+from sklearn.base import BaseEstimator, _fit_context
+from sklearn.metrics import pairwise_distances
+from sklearn.utils import check_symmetric
+from sklearn.utils._param_validation import Interval
+from sklearn.utils.extmath import svd_flip
+from sklearn.utils.validation import validate_data
+
+
+class ClassicalMDS(BaseEstimator):
+    """Classical multidimensional scaling (MDS).
+
+    This is also known as principal coordinates analysis (PCoA) or
+    Torgerson's scaling. It is a version of MDS that has exact solution
+    in terms of eigendecomposition. If the input dissimilarity matrix
+    consists of the pairwise Euclidean distances between some vectors,
+    then classical MDS is equivalent to PCA applied to this set of vectors.
+
+    Read more in the :ref:`User Guide <multidimensional_scaling>`.
```

Comment:
Looks like your previous changes to `manifold.rst` are gone?

Reply:
Wait, what? I don't see anything gone, but maybe I am looking at a wrong thing? Now I am worried. For me everything appears fine.